server/src/test/java/io/druid/server/coordinator/DruidCoordinatorTest.java

```diff
@@ -130,6 +132,9 @@ public ScheduledExecutorService create(int corePoolSize, final String nameFormat
       }
     };
     leaderAnnouncerLatch = new CountDownLatch(1);
+
+    metadataNamespaceManager = EasyMock.createNiceMock(MetadataNamespaceManager.class);
+    EasyMock.replay(metadataNamespaceManager);
```

Comment:
in order to call `EasyMock.verify(metadataNamespaceManager);` it requires a replay settings somewhere. It can probably be left out, but I put it in with the intention that it would be easier to switch to a strict mock instead of a nice mock at a later point.